app/views/config_groups/_form.html.erb

```diff
@@ -1,3 +1,5 @@
+<%= alert :header => '', :class => 'alert-warning', :close => false,
```

Comment:
I'm not sure if we need to show the warning in the edit forms as well, I think it's prominent enough in the index page